packages/openapi-types/index.ts

```diff
@@ -92,9 +97,10 @@ export namespace OpenAPIV3_1 {
   >;
 
   export type PathsObject<T extends {} = {}, P extends {} = {}> = Record<
-    string,
+    `/${string}`,
```

Comment:
⚠️ This is a breaking change! We constrain the key to be `/${string}`, as the specification states.

We need to do this because otherwise this index signature (generic `string`) would shadow the `x-${string}` signature.